src/Mvc/Mvc.Core/src/Formatters/MediaTypeCollection.cs

```diff
@@ -50,8 +50,8 @@ public void Insert(int index, MediaTypeHeaderValue item)
         /// Removes the first occurrence of a specific media type from the <see cref="MediaTypeCollection"/>.
         /// </summary>
         /// <param name="item"></param>
-        /// <returns><code>true</code> if <paramref name="item"/> is successfully removed; otherwise, <code>false</code>.
-        /// This method also returns <code>false</code> if <paramref name="item"/> was not found in the original 
+        /// <returns><c>true</c> if <paramref name="item"/> is successfully removed; otherwise, <c>false</c>.
```

Comment:
```suggestion
        /// <returns><see langword="true" /> if <paramref name="item"/> is successfully removed; otherwise, <see langword="false" />.
```